src/cmds/release.rs

```diff
@@ -31,6 +31,12 @@ pub struct ReleaseArgs {
     #[arg(long, help = "GitHub repo")]
     repo: String,
 
+    #[arg(long, help = "GitHub owners where tickets are searched. Owners can be multiple owners, separated by colon")]
+    ticket_source_owners: String,
+
+    #[arg(long, help = "GitHub repos where tickets are searched. Repos can be multiple repos, separated by colon")]
+    ticket_source_repos: String,
```

Comment:
Suggest using https://docs.rs/clap/latest/clap/struct.Arg.html#method.value_delimiter.

```suggestion
    #[arg(long, value_delimiter=",", help = "List of GitHub org/repo where issues are searched")]
    issue_source_owner_repo_pairs: Option<Vec<String>>,
```

Make this field optional, as the default org and repo will be used if not specified.

Reply:
Updated. Thank you.